hikari/users.py

```diff
@@ -120,6 +125,15 @@ class PartialUser(snowflakes.Unique, abc.ABC):
     API call.
     """
 
+    @property
+    @abc.abstractmethod
+    def _dm_channel(self) -> typing.Optional[channels.DMChannel]:
+        ...
+
+    @_dm_channel.setter
```

Comment:
Shouldn’t this be implementation detail of the users implementing this rather than a part of the interface?